crates/holochain_types/src/dna/dna_bundle.rs

```diff
@@ -36,16 +36,25 @@ impl DnaBundle {
 
     /// Convert to a DnaFile, and return what the hash of the Dna *would* have
     /// been without the provided modifier overrides
```

Comment:
Is this "without" here a typo? It's not part of the PR change but still seems relevant.

Reply:
Hmm...looks like it's not a typo, only confusing. And it seems to use the standard value for `quantum_time` and not the one from the manifest when computing the original hash...

Reply:
yes i noticed that its not using the last two fields from the IntegrityManifest.

```
/// The time used to denote the origin of the network, used to calculate
    /// time windows during gossip.
    /// All Action timestamps must come after this time.
    pub origin_time: HumanTimestamp,

    /// An array of zomes associated with your DNA.
    /// The order is significant: it determines initialization order.
    /// The integrity zome manifests.
    pub zomes: Vec<ZomeManifest>,
```

is that a bug or feature?

Reply:
and does it block the merge of this PR or merit the creation of a separate issue?

Reply:
That seems very odd to me that is uses the Holochain epoch instead of the origin time in the manifest. I think that's definitely worth an issue. No reason it should block this PR though I don't think.

I don't understand the second field, `zomes`. In what way is it not using that correctly?

Reply:
```
 pub fn to_dna_def(
        &self,
        integrity_zomes: IntegrityZomes,
        coordinator_zomes: CoordinatorZomes,
        modifiers: DnaModifiersOpt,
```

the hash is done on IntegrityZomes and CoordinatorZomes not a
` Vec<ZomeManifest>`

Reply:
I see what you're saying, yes that seems like the code at least needs cleaning up so it's working with appropriate types and not risking missing fields because it decided to deconstruct something critical.